### PR TITLE
Lesson19 hm web intake

### PR DESCRIPTION
**Pull request by SOMEWANTTOSMILE**
photo_2023-12-27_18-53-26

### CODE REVIEW

File: lesson_19_hm_web/contract.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+1.Приложение создает базу данных, а в ней таблицу со следующими колонками:
+    1. id - id пользователя и его прием пищи, он должен быть первичным ключом
+    2. user - имя пользователя, оно должно быть строковым и не быть пустым
+    3. food - название еды, оно должно быть числовым и не должно ровняться нулю
```
Comment (nicilytop-tms) [inline lesson_19_hm_web/contract.txt]: Name of food is it Number ? 🤔

File: lesson_19_hm_web/app.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class Intake(Base):
+    __tablename__ = "person_intake"
+    id = Column("id", Integer(),  primary_key=True)
+    user_name = Column("user", String(255), nullable=False, unique=True)
```
Comment (nicilytop-tms) [inline lesson_19_hm_web/app.py:18]: it doesnt number.. doesnt match the contract

File: lesson_19_hm_web/web.py
Comment (nicilytop-tms) [inline lesson_19_hm_web/web.py:1]: PEP8

**Review by nicilytop-tms — CHANGES_REQUESTED** [review]

File: lesson_19_hm_web/app.py (hunk 2 above)
Comment (SOMEWANTTOSMILE) [reply lesson_19_hm_web/app.py:18]: what exactly is not a number

File: Data_storage/json_fi.json
Comment (nicilytop-tms) [inline Data_storage/json_fi.json:1]: We dont need upload mutable files

File: lesson_19_hm_web/app.py (hunk 2 above)
Comment (nicilytop-tms) [reply lesson_19_hm_web/app.py:18]: User_name - should be a number if we look your contract..

File: Homework-lesson_9/regist_login.py
Comment (nicilytop-tms) [inline Homework-lesson_9/regist_login.py:1]: why do you commit homework - lesson 9 ?

File: Data_storage/DataStore.py
Comment (nicilytop-tms) [inline Data_storage/DataStore.py:1]: Why do you work with some Data_storage?

Rename this module.. we should call it like: `data_storage.py`

**Review by nicilytop-tms — CHANGES_REQUESTED** [review]

File: Homework-lesson_9/regist_login.py
Comment (SOMEWANTTOSMILE) [reply Homework-lesson_9/regist_login.py:1]: i rewrite contract because i did mistake

File: Data_storage/DataStore.py
Comment (SOMEWANTTOSMILE) [reply Data_storage/DataStore.py:1]: I accidentally committed old files and I don't know how to delete them

File: Homework-lesson_9/regist_login.py
Comment (SOMEWANTTOSMILE) [reply Homework-lesson_9/regist_login.py:1]: re-read the contract, the username cannot even logically be a numeric value

**Pull request closed without merging**